.github/workflows/uninstall_install_sandbox.yaml

```diff
@@ -0,0 +1,66 @@
+name: Uninstall and then re-install Traction Sandbox Instance
+
+on:
+  schedule:
+    - cron: '0 9 1,15 * *'
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  uninstall_sandbox:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Run Your Job
+        run: |
+          echo "This job runs every day at 2:00 AM PST"
+
+      - name: Authenticate and set context
+        uses: redhat-actions/oc-login@v1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
+          openshift_token: ${{ secrets.OPENSHIFT_TOKEN }}
+          namespace: ${{ secrets.OPENSHIFT_NAMESPACE }}
+      
+      - name: Uninstall Traction via Helm
+        continue-on-error: true
```

Comment:
I am not sure we want to continue on error: this could lead to a situation where the instance is not uninstalled correctly, the resources are deleted by the next step using the `oc-cli` and installation could fail.

I think it is probably best if we break on error and notify the SRE team so we can take a look (adding the notification could be the next step for this work).